static/css/mobile/header.less

```diff
@@ -181,9 +181,14 @@ header #auth-nav li {
 .site-title {
     a {
         color: #000;
+        display: block;
         font: 36px/32px @serif-stack;
+        overflow: hidden;
         text-decoration: none;
+        text-overflow: ellipsis;
         text-transform: uppercase;
+        white-space: nowrap;    
+        width: 240px;
```

Comment:
Yep - I've noted the class to use in the other comment - but sounds like a plan to make that a goal that this should truncate long strings in both mobile and desktop.